model/src/main/kotlin/io/spine/validation/ValidatedFieldView.kt

```diff
@@ -29,38 +29,35 @@ package io.spine.validation
 import io.spine.core.External
 import io.spine.core.Subscribe
 import io.spine.core.Where
-import io.spine.option.IfInvalidOption
-import io.spine.protobuf.unpack
 import io.spine.protodata.ast.Option
 import io.spine.protodata.ast.event.FieldOptionDiscovered
 
 /**
- * A view of a field that is marked with `validate`.
+ * A view of a field that is marked with `(validate)`.
+ *
+ * Note: this option [does not have][NO_ERROR_MESSAGE] an error message. It triggers
+ * in-depth validation and, in case of errors, propagates only the initial cause.
  */
 internal class ValidatedFieldView :
-    BoolFieldOptionView<ValidatedField, ValidatedField.Builder>(IfInvalidOption.getDescriptor()) {
+    BoolFieldOptionView<ValidatedField, ValidatedField.Builder>(NO_ERROR_MESSAGE) {
 
     @Subscribe
     override fun onConstraint(
         @External @Where(field = OPTION_NAME, equals = VALIDATE) e: FieldOptionDiscovered
     ) = super.onConstraint(e)
 
-    override fun saveErrorMessage(errorMessage: String) {
-        builder()!!.setErrorMessage(errorMessage)
-    }
-
     override fun enableValidation() {
         builder()!!.setValidate(true)
     }
 
-    @Subscribe
-    override fun onErrorMessage(
-        @External @Where(field = OPTION_NAME, equals = IF_INVALID) e: FieldOptionDiscovered
-    ) = super.onErrorMessage(e)
+    override fun extractErrorMessage(option: Option): String = error(
+        "Can not extract custom error message for `($VALIDATE)` option using `$option`. " +
+                "`($VALIDATE)` does not support custom error message."
```

Comment:
"... does not support custom error messages". Alternatively, if left singular, you will need an article.